multi_node_screen.sh

```diff
@@ -14,9 +15,9 @@ screen -S elixir-research -X screen -t PORT4002
 # start instances in different windows
 echo "start nodes in screen"
 screen -S elixir-research -p PORT4000 -X stuff $'PORT=4000 PERSISTENCE_PATH=apps/aecore/priv/rox_db_4000 iex -S mix phx.server -e "Aecore.Miner.Worker.resume"\n'
-screen -S elixir-research -p PORT4001 -X stuff $'sleep 5; PORT=4001 PERSISTENCE_PATH=apps/aecore/priv/rox_db_4001 iex -S mix phx.server -e "Aecore.Peers.Worker.add_peer(\\""127.0.0.1:4000\\"")"\n'
-screen -S elixir-research -p PORT4002 -X stuff $'sleep 10; PORT=4002 PERSISTENCE_PATH=apps/aecore/priv/rox_db_4002 iex -S mix phx.server -e "Aecore.Peers.Worker.add_peer(\\""127.0.0.1:4000\\"")"\n'
+screen -S elixir-research -p PORT4001 -X stuff $'sleep 3; PORT=4001 PERSISTENCE_PATH=apps/aecore/priv/rox_db_4001 iex -S mix phx.server -e "Aecore.Peers.Worker.add_peer(\\""127.0.0.1:4000\\"")"\n'
```

Comment:
@cytadela8 have you tried this, I had issues when the sleeps were the same

Reply:
Just tested again. It seams to work fine. All three nodes start, get new blocks (one mines) and are talking with each other.